vrp-cli/src/lib.rs

```diff
@@ -509,7 +514,7 @@ pub fn get_locations_serialized(problem: &Problem) -> Result<String, String> {
 }
 
 /// Gets solution serialized in json.
-pub fn get_solution_serialized(problem: Arc<CoreProblem>, config: Config) -> Result<String, String> {
+pub fn get_solution_serialized(problem: Arc<CoreProblem>, config: Config, geojson: bool) -> Result<String, String> {
```

Comment:
Not all occurrences of the function are fixed, e.g. https://github.com/reinterpretcat/vrp/blob/master/vrp-cli/tests/unit/lib_test.rs#L32

Did you try to run cargo test?